docs/preflight.md

```diff
@@ -0,0 +1,9 @@
+## Preflight Checklist
+
+This checklist is to be ran prior to kicking off the manual test plan.
+
+- [ ] Bump GoLang Vendor Dependencies
+    - [ ] AWS Go SDK
+- [ ] Review forked Dependencies for upstream security patches
```

Comment:
hmm, I'm not entirely sure how to find what dependencies were forked (and why)

Reply:
I know for example that we forked Kingpin https://github.com/gravitational/kingpin , this could be a time to review if the upstream has added the missing features.

Reply:
Anything with `source` in `Gopkg.toml` (other than kingpin, which looks like it was vendored incorrectly). This is the list:

* `github.com/gravitational/gobpf` forked from `github.com/iovisor/gobpf`
* `github.com/gravitational/go-oidc` forked from `github.com/coreos/go-oidc`
* `github.com/gravitational/logrus` forked from `github.com/sirupsen/logrus`
* `github.com/gravitational/kingpin` forked from `github.com/alecthomas/kingpin`

Last few commits in our repo usually explain why they were forked.